nuitka/distutils/DistutilCommands.py

```diff
@@ -309,10 +309,13 @@ def _build(self, build_lib):
             toml_filename = os.getenv("NUITKA_TOML_FILE")
             if toml_filename:
                 # Import toml parser like "build" module does.
-                try:
-                    from tomli import loads as toml_loads
-                except ImportError:
-                    from toml import loads as toml_loads
+                if sys.version_info >= (3, 11):
+                    from tomllib import loads as toml_loads
+                else:
+                    try:
+                        from tomli import loads as toml_loads
+                    except ImportError:
```

Comment:
We don't care about deprecations in Nuitka, since we are going to work for deprecated Python versions in the future where the deprecated thing is the only way of doing it.